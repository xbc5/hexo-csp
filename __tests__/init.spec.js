"use strict";
const { fixture, render } = require("./helpers/mock");

describe("given an invalid config", () => {
  it("should not render a CSP", async () => {
    const render = await fixture("invalid-config");
    await expect(await render("/")).toMatchSnapshot();
  });
});
