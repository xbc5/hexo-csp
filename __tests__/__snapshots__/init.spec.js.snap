// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`given an invalid config should not render a CSP 1`] = `
<!DOCTYPE html>
<html lang="en">
  <head>
    <title>
      Markdown Home
    </title>
  </head>
</html>
<body>
  <h2 id="Home">
    Home
  </h2>
</body>
`;
